java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
Do you mean typing
````
    Log.debug(...)
````
with an initial capital? Nasty trap, particularly if the IDEs autocomplete & auto-import helps make it silent. 

Can you configure the IDE to _not_ auto-import that class?